wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListContentController/ViewModel/ConversationListViewModel.swift

```diff
@@ -73,8 +73,8 @@ final class ConversationListViewModel: NSObject {
                     return "groups"
                 case .favorites:
                     return "favorites"
-                case .folder(label: let label):
-                    return label.name ?? "folder"
```

Comment:
@johnranj  I don't really get how this avoids the info to be sent to datadog?

Reply:
label.name which provides the folder name. Here the alias for the folder is just "folder" not the actual folder name.
Do you have any other suggestion for this ?

Reply:
Yes, the goal is not to send the value. But we don't have control towards field level restriction. We have only object level restriction. Alternate fix could be , returning a generic alias like other labels like favourites, groups etc.